easybuild/easyconfigs/b/Bio-DB-HTS/Bio-DB-HTS-3.01-foss-2019a-Perl-5.28.1.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PerlModule'
+
+name = 'Bio-DB-HTS'
+version = '3.01'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'https://metacpan.org/release/Bio-DB-HTS'
+description = "Read files using HTSlib including BAM/CRAM, Tabix and BCF database files"
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
I suspect this library doesn't use anything with MPI or math libraries, so I think this could be down to `GCC-8.2.0-2.31.1` ?

Reply:
Ah yes, you are right. For reasons I had and HTSlib on foss/2019a hanging around.